numpy/f2py/_backends/meson.build.template

```diff
@@ -42,5 +45,7 @@ ${source_list},
                      py_dep,
                      quadmath_dep,
 ${dep_list}
+${lib_list}
+${lib_dir_list}
```

Comment:
Actually, don't these need to be added to `dep_list`?

Reply:
Yup, but when you also pass `--dep` that populates a `dependency: ${dep_list}` right? Could you share a generated `meson.build` where you have both `--dep` (e.g. `--dep lapack`) and the new `-l -L` arguments?